src/Pallet/Pallet.cs

```diff
@@ -101,20 +103,18 @@ void OnSimulationEnded()
 		{
 			rigidBody.TopLevel = false;
 
-			rigidBody.Position = Vector3.Zero;
-			rigidBody.Rotation = Vector3.Zero;
-			rigidBody.Scale = Vector3.One;
+			rigidBody.Transform = Transform3D.Identity;
 
 			rigidBody.LinearVelocity = Vector3.Zero;
 			rigidBody.AngularVelocity = Vector3.Zero;
 
-			//Work around for #83 
+			//Work around for #S83 
```

Comment:
Not sure what the added `S` implies here. If it is accidental, please remove it.

Reply:
typo, I missed it. I will remove it.